app/models/book_request.rb

```diff
@@ -1,2 +1,9 @@
 class BookRequest < ApplicationRecord
+  belongs_to :book
+  belongs_to :borrowed_request
+
+  validates :status, numericality: {only_integer: true}
+  enum status: [:requested, :borrowing, :returned, :rejected]
```

Comment:
```suggestion
  enum status: {requested: 0, borrowing: 1, returned: 2, rejected: 3}
```
dùng hash sẽ dễ handle hơn bé nhé